versions/3.1.0.md

```diff
@@ -75,6 +75,8 @@ Path templating refers to the usage of template expressions, delimited by curly
 
 Each template expression in the path MUST correspond to a path parameter that is included in the [Path Item](#path-item-object) itself and/or in each of the Path Item's [Operations](#operation-object).
 
+The value for these path parameters MUST NOT contain any unescaped "generic syntax" characters described by [RFC3986](https://tools.ietf.org/html/rfc3986#section-3): forward slashes (`/`), forward question marks (`?`), or hashes (`#`).
```

Comment:
Do we mean the "reserved characters" defined in https://tools.ietf.org/html/rfc3986#section-2.2 or something else by '"generic syntax" characters'? And "**forward**" question-marks, that's precise!

Reply:
Reserved is more than the generic delimeters I was trying to suggest.

      reserved    = gen-delims / sub-delims

      gen-delims  = ":" / "/" / "?" / "#" / "[" / "]" / "@"

      sub-delims  = "!" / "$" / "&" / "'" / "(" / ")"
                  / "*" / "+" / "," / ";" / "="

But also maybe `[`, `]`, and `@` are ok, so I have no idea what section or word to use to describe them.